src/components/AuthContainer/AuthContainer.scss

```diff
@@ -2,8 +2,8 @@
   border-radius: 12px;
   box-shadow: 0 0 10px rgba(85, 108, 214, 0.18);
   margin: 48px auto 64px;
+  max-width: 480px;
```

Comment:
Changed width -> max width as smaller devices have the following UI:

<img width="457" alt="Screen Shot 2020-12-28 at 2 58 09 PM" src="https://user-images.githubusercontent.com/32864116/103196478-3fe94c80-491f-11eb-8960-03a70176afec.png">